pkg/db/db_local/install.go

```diff
@@ -392,10 +394,10 @@ func startServiceForInstall(port int) (*psutils.Process, error) {
 	return psutils.NewProcess(int32(postgresCmd.Process.Pid))
 }
 
-func getNextFreePort() (int, error) {
+func getNextFreePort(host string) (int, error) {
 	utils.LogTime("db_local.install.getNextFreePort start")
 	defer utils.LogTime("db_local.install.getNextFreePort end")
-	listener, err := net.Listen("tcp", "127.0.0.1:0")
+	listener, err := net.Listen("tcp", fmt.Sprintf("%s:0", host))
```

Comment:
This will try to bind the same port number on all available interfaces, is that desired?

> For TCP networks, if the host in the address parameter is empty or a literal unspecified IP address, Listen listens on *all available* unicast and anycast IP addresses of the local system.

https://pkg.go.dev/net#Listen

Reply:
Also interesting in this context:

> The address can use a host name, but this is not recommended, because it will create a listener for *at most one* of the host's IP addresses.

Reply:
As discussed in Slack, I'll update the PR to use `127.0.0.1` for all database maintenance tasks.

Reply:
Done.